components/polls/DisplayPollResults.tsx

```diff
@@ -11,194 +12,181 @@ import {
     AccordionItem,
     AccordionButton,
     AccordionPanel,
-    AccordionIcon
- } from '@chakra-ui/react'
- import { Icon } from '@chakra-ui/icon'
-
-import {Poll} from 'interfaces'
+    AccordionIcon,
+    Spinner
+} from '@chakra-ui/react'
+import { Icon } from '@chakra-ui/icon'
+import { Poll } from 'interfaces'
 import { BiBarChartSquare } from 'react-icons/bi'
 import Card from 'components/common/Card'
 import PollGraph from 'components/polls/PollGraph'
-import {useState, useEffect} from "react";
+import { useState, useEffect } from "react";
 import useStrategies from 'hooks/useStrategies'
+import { useTimer } from 'hooks/useTimer'
+import type { Duration } from 'date-fns'
 
-// import TimeGraph from 'components/polls/TimeGraph'
-
+// Component props
 type DisplayPollResultsProps = {
     pollData: Poll,
     voteData?: any,
     totalVotes?: string,
-    onCountdownComplete?: (toggle: boolean) => void
-    result?: any
 }
 
-const deltaT = (date1: string | number, date2: string | number) => {
-    let deltaT_ms = null;
-    try {
-        deltaT_ms = new Date(date1).getTime() - new Date(date2).getTime()
-    } catch {
-        deltaT_ms = 1000;
-    }
-    return deltaT_ms;
+type PollResultStack = {
```

Comment:
@tikisailor would this be relevant to the SDK you're producing?

Reply:
sdk should save us from having to manually define api types. E.g. Poll and strategy in this case.